database/postgres/database.go

```diff
@@ -233,17 +234,20 @@ func (d *PostgresDatabase) types() ([]string, error) {
 
 	var ddls []string
 	for rows.Next() {
-		var typeName, labels string
-		if err := rows.Scan(&typeName, &labels); err != nil {
+		var typeSchema, typeName, labels string
+		if err := rows.Scan(&typeSchema, &typeName, &labels); err != nil {
 			return nil, err
 		}
+		if d.config.TargetSchema != "" && d.config.TargetSchema != typeSchema {
```

Comment:
The type of TargetSchema has changed due to the modifications in https://github.com/sqldef/sqldef/pull/540. Please merge the master branch to address this change 🙏